pandas/core/generic.py

```diff
@@ -4564,7 +4563,7 @@ def pipe(self, func, *args, **kwargs):
 
     Returns
     -------
-    aggregated : %(klass)s
+    pandas.%(klass)s
```

Comment:
I don't think we prefix `Series` and `DataFrame` with `pandas`. I'd just leave with the class for consistency.